### DISCUSSION TITLE
AuthenticationProvider: TryEndAuthentication

### DISCUSSION THREAD

**Discussion by angelski** [neos-sdi/adfsmfa#219] [General]
I've installed the latest adfsmfa on WinSrv2022. I went through all the settings of adfsmfa and I just want to use MFA after the user enters user/pass.

When I enter user/pass, I get an error in the browser: Password for users_email is expired ! Access denied !
In the event viewer I get AuthenticationProvider:TryEndAuthentication Error : users_email

What am I missing?

**Comment by redhook62** [comment]
Hi @angelski 

You can look at the Security configuration side.
MMC/MFA/Security Configuration.
And Look at Password Policy.
Possibly change what is necessary or deactivate the password policy

regards
